.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```

Comment:
I've usually seen `.vscode` in `.gitignore` and this file not in source control.  I don't use that app, but I'm guessing this option would format based on each persons local settings?  Will that lead to possible format churn for someone with alternate seeings?

Reply:
I have this ticket here... https://github.com/w3c/did-core-registry/issues/16

I'd prefer to use a software linter and a pre-commit hook, that is stable wrt vs code settings, and check vs code settings into source... that way a new person using vscode has the best possible experience... I'm working towards that given I'm the one making most of the changes so far, but I'm open to changing it to whatever based on feedback.